pkg/pillar/cmd/zedmanager/zedmanager.go

```diff
@@ -212,6 +213,20 @@ func Run() {
 	ctx.subDomainStatus = subDomainStatus
 	subDomainStatus.Activate()
 
+	// Get DomainStatus from domainmgr
+	subImageStatus, err := pubsub.Subscribe("domainmgr",
+		types.ImageStatus{}, false, &ctx)
+	if err != nil {
+		log.Fatal(err)
+	}
+	subImageStatus.MaxProcessTimeWarn = warningTime
+	subImageStatus.MaxProcessTimeError = errorTime
+	subImageStatus.ModifyHandler = handleImageStatusCreate
+	subImageStatus.CreateHandler = handleImageStatusModify
+	subImageStatus.DeleteHandler = handleImageStatusDelete
```

Comment:
You don't need these empty handlers.

Reply:
Hmm.. I thought these are useful for debugging .. to see when the image was deleted.. But I agree.. May be I was overthinking.. Will remove them..